src/app/ApplicationTemplate.Shared/Client/Mock/AuthenticationEndpointMock.cs

```diff
@@ -18,9 +18,12 @@ public AuthenticationEndpointMock(IObjectSerializer serializer)
 			_serializer = serializer ?? throw new ArgumentNullException(nameof(serializer));
 		}
 
-		public async Task CreateAccount(CancellationToken ct, string email, string password)
+		public async Task<AuthenticationData> CreateAccount(CancellationToken ct, string email, string password)
 		{
 			await Task.Delay(TimeSpan.FromSeconds(2));
+
+			// Since the account creation is mocked, we need to authentify the user
```

Comment:
I don't think I understand the though behind it? Do you mean that you mimic the Login method?

Reply:
If I understand @arianeleonard , since we don't have a backend to return us the authentication data, we create the data (AccessToken, RefreshToken, Expiration) directly after the user creates an account

Reply:
@jeanplevesque there's no AuthenticationEndpoint atm in Uno App Template only AuthenticationEndpointMock